porch/controllers/rootsyncsets/pkg/controllers/rootsyncset/controller.go

```diff
@@ -199,7 +201,9 @@ func (r *RootSyncSetReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, results
 	}
 
-	return ctrl.Result{}, nil
+	jitter := time.Duration(rand.Intn(60)) * time.Second
+	pollInterval := 15*time.Minute + jitter
+	return ctrl.Result{RequeueAfter: pollInterval}, nil
```

Comment:
Did you mean to add this to the RootSyncSet controller?

Reply:
Gah!  Nice catch :-)